frontend/ui/widget/keyvaluepage.lua

```diff
@@ -441,11 +447,15 @@ function KeyValuePage:init()
         title = self.title,
         fullscreen = self.covers_fullscreen,
         width = self.width,
-        align = "left",
+        align = self.title_bar_align,
         with_bottom_line = true,
         bottom_line_color = Blitbuffer.COLOR_DARK_GRAY,
         bottom_line_h_padding = padding,
+        left_icon = self.title_bar_left_icon,
+        left_icon_tap_callback = function() self:onLeftButtonTap() end,
+        left_icon_hold_callback = function() self:onLeftButtonHold() end,
```

Comment:
I know we said "do as for Menu, for consistency" :) But:

Somehow, the `function KeyValuePage:onLeftButtonTap() -- to be overriden and implemented by the caller` we do override for ToC and Bookmark feels a bit convoluted: you have to create the Menu (toc_menu = Menu:new), then hack that toc_menu. If we want to do the simple 1-step `UIManager:show(KeyValuePage:new{...})`, we can't override them: we need to split it into 3 steps `local zzz = KeyValuePage:new{...} ; zzz:onLeftButtonTap = function()..end ; UIManager:show(zzz)`.

But here and in Menu, when we don't want/like to do it that way, we can just provide a left_icon_tap_callback that will override the need to hijack onLeftButtonTap().
So, this solution is fine, as it provides 2 ways to specify the action of the icons. One nice, and one convoluted (may be for historical reasons, the way ToC and Bookmarks were done ?). Dunno if there's any real reason to keep the convoluted one :) But ok anyway if we don't want to change too many things.

Reply:
Currently 5 modules set `left_icon_tap_callback` and 6 modules provide `onLeftButtonTap`.
For the future cleanup in favor of `left_icon_tap_callback`.